src/include/migraphx/op/logsoftmax.hpp

```diff
@@ -12,7 +12,7 @@ namespace op {
 
 struct logsoftmax
 {
-    int64_t axis = 1;
+    int64_t axis = -1;
```

Comment:
Why are we changing the default axis?

Reply:
The onnx 1.8.1 changed the default value from 1 to -1.

Reply:
Then this should be changed in the onnx parser.

Reply:
So, we will set different default values for different onnx versions, right?

Reply:
> So, we will set different default values for different onnx versions, right?

If we want backwards compatibility to 1.7 then we will probably need to do this. More importantly, onnx may change default values but that should not necessarily change our default values. Rather, the onnx parser can decide a default value that is different than ours.

Is the onnx version saved in the onnx file?